.github/workflows/sync-github-push-gha-metrics-source.yml

```diff
@@ -40,4 +51,43 @@ jobs:
         with:
           title: Update push-gha-metrics-action
           branch: chore/update-push-gha-metrics-action
-          commit-message: Update push-gha-metrics-action
\ No newline at end of file
+          commit-message: Update push-gha-metrics-action
+
+  update-snapshot:
+    # TODO: Use GATI so workflows can run automatically
```

Comment:
I see what you mean now, we can just drop this. We can consider GATI again if we have workflow requirements

Reply:
You want me to remove the `TODO`? It's also captured in [RE-2038](https://smartcontract-it.atlassian.net/browse/RE-2038).

Reply:
yeah we dont need it here since you were right, we dont have any workflows to even run on the `push` or `pullrequest` event

Reply:
e748ad4c85c03b53fd0077ff74235d838843b3e9